client/post-editor/editor-notice/index.jsx

```diff
@@ -77,20 +77,44 @@ export class EditorNotice extends Component {
 
 		switch ( key ) {
 			case 'publishFailure':
+				if ( 'post' === type ) {
+					return translate( 'Publishing of post failed' );
+				}
+
+				if ( 'page' === type ) {
+					return translate( 'Publishing of page failed' );
+				}
+
 				return translate( 'Publishing of %(typeLabel)s failed.', {
-					args: { typeLabel: typeLabel.toLowerCase() },
+					args: { typeLabel: typeLabel },
 				} );
 
 			case 'saveFailure':
 				return translate( 'Saving of draft failed.' );
 
 			case 'trashFailure':
+				if ( 'page' === type ) {
+					return translate( 'Trashing of page failed!' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Trashing of page failed.' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/lowercase-using-css-for-translation-object)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).